docs/vsphere-golden-image.md

```diff
@@ -144,10 +144,16 @@ To deploy the Windows VM as a reusable image, you have to first generalize the V
 such as installed drivers. Running the `sysprep` command with a *unattend* answer file generalizes the image and 
 makes it ready for future deployments, maintaining all the changes needed for the WMCO installation. 
 
-You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example and, it **should 
-not** be used directly. You must customize the example to fit your needs.
+A reference [unattend.xml](unattend.xml) file is provided, it **cannot** be used directly. You must customize it
+replacing the `WindowsPassword` parameter with the desired password for the Administrator user. It's recommended to
```

Comment:
`WindowsPassword` is not a parameter but the value of the `Password` and `AdministratorPassword` parameters / properties. The intent here should be to indicate to replace the `Password` and `AdministratorPassword` parameter / property values. And if they need to match, it needs to be called out here.

Reply:
Correct, `WindowsPassword` is not a parameter, is a placeholder. Thanks for calling this out, will adjust.